src/v3/src/components/Images/MobileDeviceIcon.tsx

```diff
@@ -10,29 +10,19 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import { Theme } from '@mui/material';
 import { FunctionComponent, h } from 'preact';
 
 import { loc } from '../../util';
+import { DevicesIcon } from '../Icon/OdysseyIcons';
 
 export const MobileDeviceIcon: FunctionComponent = () => (
-  <svg
-    width="16"
-    height="16"
-    viewBox="0 0 16 16"
-    fill="none"
-    xmlns="http://www.w3.org/2000/svg"
-    role="img"
-    aria-hidden="true"
-  >
-    <title>
-      {loc('icon.title.mobileBrowser', 'login')}
-    </title>
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M11.5 1H5.5C4.94772 1 4.5 1.44772 4.5 2V14C4.5 14.5523 4.94772 15 5.5 15H11.5C12.0523 15 12.5 14.5523 12.5 14V2C12.5 1.44772 12.0523 1 11.5 1ZM5.5 0C4.39543 0 3.5 0.895431 3.5 2V14C3.5 15.1046 4.39543 16 5.5 16H11.5C12.6046 16 13.5 15.1046 13.5 14V2C13.5 0.895431 12.6046 0 11.5 0H5.5ZM10 13V14H7V13H10ZM8.5 3C8.77614 3 9 2.77614 9 2.5C9 2.22386 8.77614 2 8.5 2C8.22386 2 8 2.22386 8 2.5C8 2.77614 8.22386 3 8.5 3Z"
-      fill="#1662DD"
-      class="siwIconFillPrimary"
-    />
-  </svg>
+  <DevicesIcon
+    sx={(theme: Theme) => ({
+      color: theme.palette.primary.main,
+      width: '16px',
+      height: '16px',
+    })}
+    titleAccess={loc('icon.title.mobileBrowser', 'login')}
```

Comment:
I see Odyssey now combines both desktop and mobile into one image, possibly a question for @alexdahl-okta; Should we create a new title to appropriately match the new icon? Or should we continue with the separate text depending if they are on Mobile vs desktop? 

When on mobile the text would read "Mobile browser" whereas on desktop it will read "Browser" 
This will be the new icon: 
<img width="141" alt="image" src="https://github.com/okta/okta-signin-widget/assets/97472729/42bfaf10-4e61-42af-a501-0b860c71785d">

Reply:
Great question.

My hunch is to combine, and just say "browser" for both. The distinction between mobile and desktop was already subtle, and I don't think that context adds much value.

Do we know what the values for the browser context are like?

I recall that they're something like "Chrome on MAC_OS_X"... If they're likely to include the operating system, then that'd make it even safer to remove the context from the icon label.

Reply:
Yep! For mobile, it would say something like: `FIREFOX on Android` or `Safari on iOS`. Whereas on desktop, it would say `FIREFOX on Mac OS X`. 

That makes sense to consolidate it to browser since more context is provided in the label. 

@denysoblohin-okta Would it be possible to get rid of the `MobileDeviceIcon.tsx` and then in [here](https://github.com/okta/okta-signin-widget/blob/master/src/v3/src/transformer/terminal/transformEmailMagicLinkOTPOnlyElements.ts#L97) you can just reference the combined DevicesIcon (which has the browser title).